tests/testthat/test-shinytest2-tm_a_pca.R

```diff
@@ -0,0 +1,153 @@
+testthat::test_that("e2e - tm_a_pca: data extract changes output", {
+  skip_if_too_deep(5)
+
+  require(shinytest2)
```

Comment:
Temporary requirement from a `TealAppDriver` call to `platform_variant` (this need to be fixed upstream)

Reply:
https://github.com/insightsengineering/teal/pull/1198

Reply:
Thanks @m7pr :100: